packages/common/auth/src/types/authmaterial.ts

```diff
@@ -1,10 +1,28 @@
 import type { SDKExternalUser } from "./user";
 
+export type AuthMaterialBasic = {
+    jwtToken?: string;
+    refreshToken: string;
+};
+
+interface refreshTokenObject {
+    secret: string;
+    expiresAt: string;
+}
+
 export type AuthMaterial = {
     jwtToken: string;
-    refreshToken: {
-        secret: string;
-        expiresAt: string;
-    };
+    refreshToken: refreshTokenObject;
     user: SDKExternalUser;
 };
+
+export type AuthMaterialResponse = {
+    jwt: string;
+    refresh: refreshTokenObject;
+    user: SDKExternalUser;
+};
+
+export type AuthSession = {
+    jwtToken: string;
```

Comment:
`jwtToken` on the other hand I don't like haha. Isn't the `t` "token" anyway? I think just `jwt` here and other places for consistency
```suggestion
    jwt: string;
```